src/pythie_serving/utils.py

```diff
@@ -62,7 +63,17 @@ def get_np_dtype(tf_type: types_pb2.DataType):
         raise TypeError(f"Could not infer numpy type for {tf_type}")
 
 
-def make_tensor_proto(values: List[Any]):
+def get_csv_type(type_mapping: Dict[str, str]):
```

Comment:
Return type is missing
Maybe we should add mypy + pyupgrade to the gitlab CI ?

Reply:
it's a bit complicated to run mypy because we type with also use the protobufs that are defined in the repo for typing. For it to work, we have to generate the corresponding stubs.

I'll add a issue to tackle this